examples/hh/configs/ppo_hh.yml

```diff
@@ -0,0 +1,57 @@
+train:
+  seq_length: 1024
+  epochs: 10000
+  total_steps: 10000
+  batch_size: 4
+
+  checkpoint_interval: 10000
+  eval_interval: 500
+
+  pipeline: "PromptPipeline"
+  trainer: "AcceleratePPOTrainer"
+  checkpoint_dir: "checkpoints/ppo_hh"
+
+model:
+  model_path: "EleutherAI/gpt-j-6B"
```

Comment:
Maybe let's make the default example use a smaller model so more people can run it